app/models/data/local_authority_user.rb

```diff
@@ -3,29 +3,27 @@ class LocalAuthorityUser
     include ToCsv
 
     # @return [String]
-    def self.to_csv
-      new.generate_csv
+    def self.column_names
+      ['Local Authority', 'Users']
     end
 
-    # @return [String]
-    def generate_csv
-      CSV.generate do |csv|
-        csv << ['Local Authority', 'Users']
-        count_by_local_authority.each do |local_authority, count|
-          csv << [local_authority, count]
-        end
+    # @return [Array<Array>]
+    def self.dashboard
+      data = Hash.new { |hash, key| hash[key] = [] }
```

Comment:
Am I right in thinking this data store is common to a few bespoke models and is being used so we can select values for coercion. That could justify it getting pulled out in to the module and that use case documented.